src/app/components/CommentCard/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import { Comment } from '@interfaces/comment';
+
+import styles from './styles';
+
+interface Props extends Comment {}
+
+function CommentCard({ avatar, author, content }: Props) {
```

Comment:
Move this component inside `CommentList/components` or are you going to use it from another screen? Same with `BookCard` => `app/screens/BookList/components/BookCard`. Both components will only be used in one screen.